requirements.txt

```diff
@@ -1,21 +1,23 @@
-git+https://github.com/metabrainz/brainzutils-python.git@v1.12.0
+git+https://github.com/metabrainz/brainzutils-python.git@v1.15.0
 click==6.7
-Flask-Admin==1.5.3
-Flask-Babel==0.11.2
+Werkzeug==1.0.1
+Flask-Admin==1.5.7
+Flask-Babel==2.0.0
 Flask-Login==0.4.1
 Flask-SQLAlchemy==2.4.0
-Flask-Testing==0.7.1
-Flask-Uploads==0.2.1
-Flask-WTF==0.14.2
-psycopg2==2.7.3
-pytest-cov==2.5.1
-pytest==3.2.1
+Flask-Testing==0.8.1
+-e git://github.com/maxcountryman/flask-uploads.git#egg=Flask_Uploads
```

Comment:
did you need to update this to a newer version because 0.2.1 doesn't support something? It'd be good to add an explicit tag or commit here to make sure that random changes to master don't break the build: 
`-e git://github.com/maxcountryman/flask-uploads.git@commit-hash#egg=Flask_Uploads`